src/main/java/com/example/java/test/junior/developer/configuration/WebClientConfiguration.java

```diff
@@ -21,4 +22,9 @@ public WebClient keycloakWebClient() {
         .baseUrl(tokenUrl)
         .build();
   }
+
+  @Bean
+  public RestTemplate restTemplate() {
```

Comment:
Please, don't make me repeat same things 1000 times. We. Don't. Use. RestTemplate. We. Use. WebClient only. Delete this.